src/Routes/HomePage/index.js

```diff
@@ -91,7 +91,7 @@ const HomePage = () => {
               </GridItem>
               <GridItem md={6} sm={12}>
                 <iframe
-                  style={{ width: '100%', height: '315px' }}
```

Comment:
Since you changed this line, could be better to use `className` instead of direct styles.